### ISSUE TITLE
def/Zealot Rush design question

### ISSUE THREAD

**Issue by PointMeAtTheDawn** [Ar3sgice/scmaps#1]
Hello! Btw, you should know that Zealot Rush is still a popularly hosted map in 2022 on US West ^^.

TL;DR: I was curious why you decoupled enemy health from wave #. Wave # primarily determines spawn rate (along with leg enhancements / hero upgrade), but starting enemy shields and health are instead tied to kills.

(Dry reference #s to jog your memory follow):
- Starts at 0% shields 10% health, increments by 10% each every 1500 kills.
- Progression unimpacted by swap from regular zealot to Fenix (which quintuples maxes from 1k to 5k each, and bumps armor from 0 to 200) on wave 6.
- This means strats that maximally bottleneck the spawn area significantly ease the midgame difficulty ramp by slowing health progression.
- Waves are primarily tracked with a death counter, but wave 5 (speed up) is an exception, so waves 5-10 are 1 off on their death counter. Perhaps this foible is related to the intent?

Hope all is well :].
